dm/tests/_utils/wait_process_exit

```diff
@@ -4,7 +4,7 @@
 process=$1
 
 WAIT_COUNT=0
-while [ $WAIT_COUNT -lt 30 ]; do
+while [ $WAIT_COUNT -lt 120 ]; do
```

Comment:
```
[2022/05/28 10:27:52.614 +08:00] [INFO] [main.go:84] ["got signal to exit"] [signal=hangup]
[2022/05/28 10:27:52.614 +08:00] [INFO] [server.go:281] ["closing server"]
[2022/05/28 10:27:52.614 +08:00] [INFO] [election.go:325] ["current member is not the leader"] [component=election] ["current member"="{\"id\":\"master1\",\"addr\":\"127.0.0.1:8261\"}"] [leader="{\"id\":\"master3\",\"addr\":\"127.0.0.1:8461\"}"]
[2022/05/28 10:27:52.614 +08:00] [WARN] [election.go:359] ["ignore notify the leader's information because context canceled"] [component=election] ["current member"="{\"id\":\"master1\",\"addr\":\"127.0.0.1:8261\"}"] [leader="{\"id\":\"master3\",\"addr\":\"127.0.0.1:8461\"}"]
[2022/05/28 10:27:52.615 +08:00] [INFO] [election.go:190] ["election is closing"] [component=election] ["current member"="{\"id\":\"master1\",\"addr\":\"127.0.0.1:8261\"}"]
[2022/05/28 10:27:52.616 +08:00] [DEBUG] [watch.go:191] ["failed to receive watch request from gRPC stream"] [component="embed etcd"] [error="rpc error: code = Canceled desc = context canceled"]
[2022/05/28 10:27:52.620 +08:00] [DEBUG] [store.go:1133] ["found command name"

[2022/05/28 10:28:52.783 +08:00] [DEBUG] [peer_status.go:76] ["peer deactivated again"] [component="embed etcd"] [peer-id=7315b5b7a5c31dc] [error="failed to dial 7315b5b7a5c31dc on stream MsgApp v2 (dial tcp 127.0.0.1:8292: connect: connection refused)"]
[2022/05/28 10:28:52.852 +08:00] [DEBUG] [interceptor.go:182] ["request stats"] [component="embed etcd"] ["start time"=2022/05/28 10:28:49.130 +08:00] ["time spent"=3.722337167s] [remote=127.0.0.1:50042] ["response type"=/etcdserverpb.Lease/LeaseRevoke] ["request count"=-1] ["request size"=-1] ["response count"=-1] ["response size"=-1] ["request content"=]
[2022/05/28 10:28:52.853 +08:00] [ERROR] [election.go:205] ["fail to close etcd session"] [component=election] [lease=152419835867964164] [error="context deadline exceeded"]
[2022/05/28 10:28:52.853 +08:00] [INFO] [election.go:198] ["election is closed"] [component=election] ["current member"="{\"id\":\"master1\",\"addr\":\"127.0.0.1:8261\"}"]
[2022/05/28 10:28:52.854 +08:00] [INFO] [etcd.go:368] ["closing etcd server"] [component="embed etcd"] [name=master1] [data-dir=default.master1] [advertise-peer-urls="[https://127.0.0.1:8291]"] [advertise-client-urls="[https://127.0.0.1:8261]"]
[2022/05/28 10:28:52.856 +08:00] [INFO] [server.go:1433] ["skipped leadership transfer; local server is not leader"] [component="embed etcd"] [local-member-id=8bbb35d21267021d] [current-leader-member-id=0]
[2022/05/28 10:28:52.856 +08:00] [INFO] [peer.go:330] ["stopping remote peer"] [component="embed etcd"] [remote-peer-id=5e5ef2613797764c]
[2022/05/28 10:28:52.857 +08:00] [INFO] [stream.go:294] ["stopped TCP streaming connection with remote peer"] [component="embed etcd"] [stream-writer-type="stream MsgApp v2"] [remote-peer-id=5e5ef2613797764c]
[2022/05/28 10:28:52.857 +08:00] [INFO] [stream.go:294] ["stopped TCP streaming connection with remote peer"] [component="embed etcd"] [stream-writer-type="stream Message"] [remote-peer-id=5e5ef2613797764c]
[2022/05/28 10:28:52.857 +08:00] [INFO] [pipeline.go:85] ["stopped HTTP pipelining with remote peer"] [component="embed etcd"] [local-member-id=8bbb35d21267021d] [remote-peer-id=5e5ef2613797764c]
[2022/05/28 10:28:52.857 +08:00] [INFO] [stream.go:442] ["stopped stream reader with remote peer"] [component="embed etcd"] [stream-reader-type="stream MsgApp v2"] [local-member-id=8bbb35d21267021d] [remote-peer-id=5e5ef2613797764c]
[2022/05/28 10:28:52.858 +08:00] [INFO] [stream.go:442] ["stopped stream reader with remote peer"] [component="embed etcd"] [stream-reader-type="stream Message"] [local-member-id=8bbb35d21267021d] [remote-peer-id=5e5ef2613797764c]
[2022/05/28 10:28:52.858 +08:00] [INFO] [peer.go:335] ["stopped remote peer"] [component="embed etcd"] [remote-peer-id=5e5ef2613797764c]
[2022/05/28 10:28:52.858 +08:00] [INFO] [peer.go:330] ["stopping remote peer"] [component="embed etcd"] [remote-peer-id=7315b5b7a5c31dc]
[2022/05/28 10:28:52.858 +08:00] [INFO] [stream.go:294] ["stopped TCP streaming connection with remote peer"] [component="embed etcd"] [stream-writer-type="stream MsgApp v2"] [remote-peer-id=7315b5b7a5c31dc]
[2022/05/28 10:28:52.858 +08:00] [INFO] [stream.go:294] ["stopped TCP streaming connection with remote peer"] [component="embed etcd"] [stream-writer-type="stream Message"] [remote-peer-id=7315b5b7a5c31dc]
[2022/05/28 10:28:52.858 +08:00] [INFO] [pipeline.go:85] ["stopped HTTP pipelining with remote peer"] [component="embed etcd"] [local-member-id=8bbb35d21267021d] [remote-peer-id=7315b5b7a5c31dc]
[2022/05/28 10:28:52.859 +08:00] [INFO] [stream.go:442] ["stopped stream reader with remote peer"] [component="embed etcd"] [stream-reader-type="stream MsgApp v2"] [local-member-id=8bbb35d21267021d] [remote-peer-id=7315b5b7a5c31dc]
[2022/05/28 10:28:52.859 +08:00] [INFO] [stream.go:442] ["stopped stream reader with remote peer"] [component="embed etcd"] [stream-reader-type="stream Message"] [local-member-id=8bbb35d21267021d] [remote-peer-id=7315b5b7a5c31dc]
[2022/05/28 10:28:52.859 +08:00] [INFO] [peer.go:335] ["stopped remote peer"] [component="embed etcd"] [remote-peer-id=7315b5b7a5c31dc]
[2022/05/28 10:28:52.921 +08:00] [INFO] [etcd.go:563] ["stopping serving peer traffic"] [component="embed etcd"] [address=127.0.0.1:8291]
[2022/05/28 10:28:52.923 +08:00] [INFO] [etcd.go:568] ["stopped serving peer traffic"] [component="embed etcd"] [address=127.0.0.1:8291]
[2022/05/28 10:28:52.923 +08:00] [INFO] [etcd.go:370] ["closed etcd server"] [component="embed etcd"] [name=master1] [data-dir=default.master1] [advertise-peer-urls="[https://127.0.0.1:8291]"] [advertise-client-urls="[https://127.0.0.1:8261]"]
[2022/05/28 10:28:52.923 +08:00] [INFO] [server.go:283] ["server closed"]
[2022/05/28 10:28:52.923 +08:00] [INFO] [main.go:91] ["dm-master exit"]
```

30s is not enough for kill dm-master it takes 60s 

From the logs, we can see that it may take up to 60 s to fully exit dm-master

Reply:
OK we can take a look later